src/strbuf.c

```diff
@@ -437,35 +434,35 @@ BOOL Strbuf_IsTrainerName(Strbuf *strbuf)
 void Strbuf_ConcatTrainerName(Strbuf *dst, Strbuf *src)
 {
     if (Strbuf_IsTrainerName(src)) {
-        Charcode *dstC = &dst->data[dst->size];
-        Charcode *srcC = &src->data[1];
+        charcode_t *dstChar = &dst->data[dst->size];
+        charcode_t *srcChar = &src->data[1];
         s32 bit = 0;
-        s32 outsize = 0;
-        Charcode cur = 0;
+        s32 charsAdded = 0;
+        charcode_t curChar = 0;
 
         while (TRUE) {
-            cur = (*srcC >> bit) & COMPRESSED_MASK;
+            curChar = (*srcChar >> bit) & COMPRESSED_MASK;
             bit += 9;
```

Comment:
not actually sure what bit is, but maybe comments to explain it would be better. (it might be too abstract to explain in the variable name). you can always add a `// TODO` comment if you don't want to comment it now so the PR can be merged.